impeller/renderer/backend/vulkan/context_vk.cc

```diff
@@ -130,13 +131,16 @@ void ContextVK::Setup(Settings settings) {
 
   raster_message_loop_ = fml::ConcurrentMessageLoop::Create(
       std::min(4u, std::thread::hardware_concurrency()));
-#ifdef FML_OS_ANDROID
   raster_message_loop_->PostTaskToAllWorkers([]() {
+    // Currently we only use the worker task pool for small parts of a frame
+    // workload, if this changes this setting may need to be adjusted.
+    fml::RequestAffinity(fml::CpuAffinity::kNotPerformance);
+#ifdef FML_OS_ANDROID
```

Comment:
In a later patch, we should also move these set priority calls into a base library. We need to do the same for other platforms too (though I guess we don't care about Vulkan on macOS). Just seems odd to have platform specific ifdefs in platform agnostic code.